core/shim.js

```diff
@@ -85,3 +85,14 @@ exports.createEvent = function(thread, senderId, senderName, message) {
 		body: message
 	};
 };
+
+exports.createEventHelpers = function(event) {
+    event.arguments = event.body.match(/(?:[^\s"]+|"[^"]*")+/g);
+    if (event.arguments === null) {
+        event.arguments = [''];
+    }
+    event.arguments_body = event.body.substr(event.arguments[0].length + 1);
+    for (var i = 0; i < event.arguments.length; i++) {
+        event.arguments[i] = event.arguments[i].replace(/(^["])|(["]$)/g, '');
+    }
+};
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529926658/issues/source?bid=3282987&fileBranchId=3282998#l98)